.github/workflows/docker.yml

```diff
@@ -5,7 +5,7 @@ on:
       - main
     tags:
       - v[0-9]+.[0-9]+.[0-9]+
-
+permissions: none
```

Comment:
Question: how does the `actions/checkout@v2` on L15 work without `contents: read`? I expected that to be via the automatically generated `GITHUB_TOKEN`.

Reply:
Ah yes, I think you're right, will set `contents:read`